app/templates/review_item/accept.hbs

```diff
@@ -24,7 +23,6 @@
                         content=subPackageTypes
                         on-change=(action (mut pkg.packageTypeId))
                         selectedValue=pkg.packageType
-                        prompt=(t 'placeholder.package_type')
```

Comment:
What is the reason for removing placeholder?

Reply:
As  required we need to show default sub package type, so every package has default sub package type , if not package type it self is default package type so it will show default sub package type.